tidb-cloud/data-service-manage-data-app.md

```diff
@@ -13,7 +13,7 @@ This document describes how to manage your Data Apps in the TiDB Cloud console.
 
 To create a Data App for your project, perform the following steps:
 
-1. On the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project, click <MDSvgIcon name="icon-create-data-app" /> **Create DataApp** in the left pane.
+1. On the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project, click **Create Data App**.
```

Comment:
![image](https://github.com/pingcap/docs/assets/79440533/f90ed142-e9a2-4922-8456-f387c4c4accd)
@lilin90 @Oreoxmt PTAL

Reply:
@qiancai Looks like if a user comes to this page for the first time, the display is different?
<img width="1480" alt="image" src="https://github.com/pingcap/docs/assets/30922556/3f92b0b2-2a0e-4a15-b2f1-1c44ac5fc6a2">

Reply:
Yes, I've updated the docs in https://github.com/pingcap/docs/pull/13872/commits/ba466204d9aebd918f3a8691b1d0d5e82ee01d89. PTAL